snaps_k8s/provision/ansible_configuration.py

```diff
@@ -281,6 +275,37 @@ def launch_multus_cni(k8s_conf):
         })
 
 
+def create_cluster_role(k8s_conf):
+    """
+    This function is used to launch multus cni
+    """
+    master_host_name, master_ip = config_utils.get_first_master_host(k8s_conf)
+    logger.info('EXECUTING CREATE CLUSTER ROLE PLAY. Master ip - %s, '
+                'Master Host Name - %s', master_ip, master_host_name)
+    ret_val = ansible_utils.apply_playbook(consts.K8_MULTUS_SET_MASTER,
+                                           [master_ip], consts.NODE_USER)
+    if ret_val:
+        raise Exception('Failed in EXECUTING cluster role file creation')
+
+    logger.info('EXECUTING MASTER cluster role define')
+    node_configs = config_utils.get_node_configs(k8s_conf)
+    if node_configs and len(node_configs) > 0:
+        for node_config in node_configs:
+            host = node_config[consts.HOST_KEY]
+            pb_vars = {'hostname': host[consts.HOSTNAME_KEY]}
+            ret_val = ansible_utils.apply_playbook(
+                consts.K8_MULTUS_CLUSTER_ROLE_DEFINE, [master_ip],
+                consts.NODE_USER, variables=pb_vars)
+            if ret_val:
+                raise Exception('Failed in  cluster role define')
+
+    logger.info('EXECUTING cluster role creation')
+    ret_val = ansible_utils.apply_playbook(
```

Comment:
No return value is ever returned. Exception will be raised if there is a problem and the check below will always fail. This is what is causing the unit test to fail.